### ISSUE TITLE
[Game]: We Happy Few

### ISSUE THREAD

**Issue by pooflabber** [ItsLogic/Xbox-Achievement-Unlocker#294]
### Confirmation

- [X] I have verified that this is not a duplicate request.

### Game Link

https://www.trueachievements.com/game/We-Happy-Few/achievements

### Game Platform

Console and PC

### Additional Game Information

Priority is this achievement (https://www.trueachievements.com/a257379/hot-on-her-heels-achievement) as its very buggy even playing the game multiple times trying to unlock this wont work for majority of players.

**Comment by itsnotlogic** [comment]
I have tried to get the data when I unlocking an achievement on this game on PC but nothing in Wireshark.
Did someone tried also ?

**Comment by ghost** [comment]
Yes l do want that too as of now it is there but all achievement are locked 

L try it so broken

**Comment by VeryOriginalUsername2025** [comment]
bro wtf? if you want them achievements so badly go for it and make the json yourself and post it in the discord channel. ALL games are in the tool, but event based are not that easy to do..

**Comment by ghost** [comment]
Look bro l 100% support what they are doing but like everyone l want them to know the game we want and l said l 100 love what they are doing ok

**Comment by VeryOriginalUsername2025** [comment]
I just saw it's already done via https://github.com/Fumo-Unlockers/Xbox-Achievement-Unlocker/pull/351

you just need your xbox token and you can unlock them all.

**Comment by ghost** [comment]
l am new so can you explain pls how to get them xbox token

**Comment by VeryOriginalUsername2025** [comment]
I advice you to join the discord server. there are two tutorials.

**Comment by ghost** [comment]
OK well thx you and have a good day or night

**Comment by VeryOriginalUsername2025** [comment]
@ItsLogic can be closed. @Victor7095 made it fully supported and it has been merged in #351
